### PR TITLE
closes #84

### LINKED ISSUE #84: Couple of table improvements
I'm thinking that we remove date_time from the events model and add an individual date and time column. That way it's easier to fill in.

Also We should change location to venue in the events model as well.

### PR DESCRIPTION
**Pull request by cubadomingo**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -9,6 +9,8 @@ def show
 
   def new
     @event = Event.new
+    @categories = Category.all
+    @subregions = @region.subregions
```
Comment (tmobaird) [inline app/controllers/events_controller.rb]: you don't have to set `@subregions` if you don't want. You can simply make those calls in the views if you'd like

**Comment by tmobaird** [comment]
I think the coverage dropped 20% because of the failed tests

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (cubadomingo) [inline app/controllers/events_controller.rb]: Oh yeah huh just @regions.subregions.each

**Comment by cubadomingo** [comment]
Yeah bro, it failed because of invalid factories. I need to change location to venues.

**Comment by tmobaird** [comment]
:+1:

**Pull request merged**